.github/workflows/release.yml

```diff
@@ -32,6 +32,10 @@ jobs:
       - run: cd ./packages/web-component && npm version patch
       - run: cd ./packages/ngx-web-component && npm version patch
       - run: npx nx build web-component && npx nx bundle web-component && npx nx build ngx-web-component
+        # We have to do this now because nx is broken (see https://github.com/nrwl/nx/issues/19989)
+      - run: |
```

Comment:
gah - that's unfortunate.

Reply:
yeah.  my general irritation with nx maybe is showing a bit too much :)